workflows/eager.nf

```diff
@@ -128,6 +129,13 @@ workflow EAGER {
         if ( params.preprocessing_tool == 'fastp' && !adapterlist.extension.matches(".*(fa|fasta|fna|fas)") ) error "[nf-core/eager] ERROR: fastp adapter list requires a `.fasta` format and extension (or fa, fas, fna). Check input: --preprocessing_adapterlist ${params.preprocessing_adapterlist}"
     }
 
+    // QualiMap
+   if ( params.snpcapture_bed ) {
+    ch_snpcapture_bed = Channel.fromPath(params.snpcapture_bed, checkIfExists: true).collect()
```

Comment:
`ch_snpcapture_bed` is currently generated from the path without any `meta` map. Later on when you do [channel manipulations](https://github.com/nf-core/eager/blob/89776d88987d086ab9e7f8114c17f2b6580fa004/workflows/eager.nf#L365-L372), a map is expected.
You will need a `meta` here to test the module properly. You could do channel manipulations to take the `meta` of the channel with the reference fasta.
With the addition of the reference-sheet, the loading of the snpcapture_bed will be overhauled.